relay-server/src/actors/envelopes.rs

```diff
@@ -1533,7 +1533,10 @@ struct ProcessMetrics {
     pub items: Vec<Item>,
 
     /// The target project.
-    pub project: Addr<Project>,
+    pub public_key: ProjectKey,
+
+    /// The project cache.
+    pub project_cache: Addr<ProjectCache>,
```

Comment:
This is OK in the interim. In the long run, this has to be changed in one of two ways:
1. Make `EnvelopeManager`, `ProjectCache` global singletons (system actors). 
2. Remove either the dependency from `ProjectCache` to `EnvelopeManager` or vice versa.